db/migrate/20210525093549_add_parent_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddParentToUser < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :parent_id, :bigint,index:true
```

Comment:
@rubyboy-hub  You can have add_reference to add it as foreign key. I see that you have used reference in some places but not in some places.

Reply:
added reference for parent_id